demo/data/SIL_OPEN_FONT_LICENSE.txt

```diff
@@ -0,0 +1,93 @@
+Copyright 2010, 2012 Adobe Systems Incorporated (http://www.adobe.com/), with Reserved Font Name 'Source'. All Rights Reserved. Source is a trademark of Adobe Systems Incorporated in the United States and/or other countries.
```

Comment:
Let's add a line at the top of the file that states that this license applies to the SourceCodePro-Regular.ttf file.

Reply:
Yea that might help a little more to resolve confusion as a oneline since SIL basically applies to the name of the font so there might not be a ton of mods to the thing in the future without stating that a certain char was changed, added etc, etc...